gadget-container/gadget-from-local-bin.Dockerfile

```diff
@@ -10,6 +10,7 @@ RUN set -ex; \
 COPY files/runc-hook-prestart.sh /bin/runc-hook-prestart.sh
 COPY files/runc-hook-poststop.sh /bin/runc-hook-poststop.sh
 COPY files/entrypoint.sh /entrypoint.sh
+COPY cleanup.sh /cleanup.sh
```

Comment:
Oops, the previous lines have `files/` and it is not the right directory anymore. But that's unrelated to your patch.

Reply:
I fixed the `files/` lines in https://github.com/kinvolk/inspektor-gadget/pull/71